plugins/cilium-cni/main.go

```diff
@@ -397,6 +397,8 @@ func cmdAdd(args *skel.CmdArgs) (err error) {
 		return fmt.Errorf("unable to connect to Cilium daemon: %s", client.Hint(err))
 	}
 
+	conf, err := getConfigFromCiliumAgent(c)
+
```

Comment:
you moved the line bot not the err check

Reply:
Ah... Updated.